test/unit/services/DateService.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { DateService, TimespanResolution } from "services/DateService";
+
+
+type TimeArray = [
+  days: number,
+  hours: number,
+  minutes: number,
+  seconds: number
+];
+type TestData = [
+  resolution: TimespanResolution,
+  timeArray: TimeArray,
+  expected: string
+];
+
+describe("DateService", () => {
+  let dateService: DateService;
+  beforeAll(() => {
+    dateService = new DateService();
+  });
+
+  describe("#ticksToTimeSpanString", () => {
+    /* prettier-ignore */
+    const testData: TestData[] = [
+      [TimespanResolution.largest2, [1, 1, 1, 1], "1 day, 1 hour"],
+      [TimespanResolution.largest2, [1, 1, 1, 0], "1 day, 1 hour"],
+      [TimespanResolution.largest2, [1, 1, 0, 1], "1 day, 1 hour"],
+      [TimespanResolution.largest2, [1, 1, 0, 0], "1 day, 1 hour"],
+      [TimespanResolution.largest2, [1, 0, 1, 1], "1 day, 1 minute"],
+      [TimespanResolution.largest2, [1, 0, 1, 0], "1 day, 1 minute"],
+      [TimespanResolution.largest2, [1, 0, 0, 1], "1 day, 1 second"],
+      [TimespanResolution.largest2, [1, 0, 0, 0], "1 day, 0 seconds"],
+      [TimespanResolution.largest2, [0, 1, 1, 1], "1 hour, 1 minute"],
+      [TimespanResolution.largest2, [0, 1, 1, 0], "1 hour, 1 minute"],
+      [TimespanResolution.largest2, [0, 1, 0, 1], "1 hour, 1 second"],
+      [TimespanResolution.largest2, [0, 1, 0, 0], "1 hour, 0 seconds"],
+      [TimespanResolution.largest2, [0, 0, 1, 1], "1 minute, 1 second"],
+      [TimespanResolution.largest2, [0, 0, 1, 0], "1 minute, 0 seconds"],
```

Comment:
@dkent600 
We talked about it a bit, but here more formalized:
What should be the desired output for this case? (where amount of milliseconds is exactly 1 minute)

Imo, to only have minimal changes, we can omit this corner case